src/components/Carousel.js

```diff
@@ -614,6 +614,7 @@ export default class Carousel extends Component {
    * @return {ReactElement} element
    */
   renderArrowLeft = () => {
+    const { value } = this.props;
```

Comment:
Value should not be retrieved directly from props. Use the `this.getCurrentValue()` method for this as you did in `renderArrowRight` method.